azext_iot/digitaltwins/commands_jobs.py

```diff
@@ -0,0 +1,41 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azext_iot.digitaltwins.providers.import_job import ImportJobProvider
+from knack.log import get_logger
+from uuid import uuid4
+
+logger = get_logger(__name__)
+
+
+def create_import_job(
+    cmd, name_or_hostname: str, data_file_name: str, input_blob_container_name: str,
+    input_storage_account_name: str, output_file_name: str, output_blob_container_name: str = None,
```

Comment:
can we give a default file name for the output_file_name? Something like data_file_name (with ndjson stripped) + output.ndjson